### PR TITLE
New humanoid hostile mobs

### PR DESCRIPTION
**Pull request by unid15**
- Added vox hostile simple mobs: they require nitrogen to survive, but thankfully they can be spawned with a spacesuit
- Added vox sharpshooters - hostile simple mobs equipped with crossbows and vox quills. Their attacks are very powerful, but the cooldown between them is high
- Added vox cyber operators - hostile simple mobs equipped with ion guns
- Added skellington hostile simple mobs: they have 50 health (other humanoids have 100)
- Added skellington corsairs - hostile simple mobs equipped with a sword
- Added skellington cannoneers - hostile simple mobs equipped with laser cannons
- Added cultist simple mobs
- Added champions of Nar-Sie - hostile simple mobs equipped with a cult sword
- Added priests of Nar-Sie - hostile simple mobs that smite and deafen their enemies at range, and summon EMPs
- Added tajaran simple mobs
- Added diona simple mobs - they're very slow but have a lot of health and slowly regen (because they're simple mobs, they don't check for light like real dionae do)
- Added diona spearmen - dionae armed with a spear
- Added diona chemists - dionae with a chemmask, chemical pack, labcoat and two hatchets. They are incredibly quick and tanky
- Added grey simple mobs - naked and in spacesuits
- Added mime simple mobs - they work like clown mobs (so they only attack in retaliation)
- Corpse landmarks can now spawn mutant races (vox, grays, ...)
- Greys can now be spawned with the "spawn" command (the path is /mob/living/carbon/human/grey)
- Added magenta icon_state to floors.dmi (just a 32x32 magenta square)

### CODE REVIEW

**Comment by ghost** [comment]
Was previously doing a bus involving a cult temple, and of note is the lack of cult hostile mobs that can be spawned.

Any chance of there being some humanoid cultist mobs, like one type that wields the cult sword, another that uses certain talisman effects like EMP?

**Comment by unid15** [comment]
Some more mobs
priest of Nar-Sie, champion of Nar-Sie, cultist

Priest of Nar-Sie can deafen or smite (10-20 brute) his enemies at range, or summon a 3-5-7 EMP
Cultist and Champion of Nar-Sie are just melee hostile mobs (Champion does 20-35 damage, cultist does 2-7)

Thanks for the suggestion

**Comment by Imagebard** [comment]
This PR is beautiful.

**Comment by Intigracy** [comment]
Can you make it so hostile mobs will attack you until you're dead, rather than just to crit?

**Comment by unid15** [comment]
That's a big balance change @Intigracy . You can create a PR that changes this var to 1 if you want all hostile mobs to attack you until you're dead:

https://github.com/d3athrow/vgstation13/blob/Bleeding-Edge/code/modules/mob/living/simple_animal/hostile/hostile.dm#L27

**Comment by unid15** [comment]
last mobs for now


- slow and robust dionae and diona spearmen
- very fast, robust and regenerating diona chemists (they also dual wield hatchets)
- catbeasts :cat:

100% tested and ready to merge

**Comment by Intigracy** [comment]
oh my god that mankini catbeast.

**Comment by ghost** [comment]
Somebody a bit ago was asking for an alternate to the retaliate clown in the form of a retaliate mime for their vault

**Comment by Shadowmech88** [comment]
Do the diona chemists actually have chems in their packs, or is their tankiness due to their own vars?

**Comment by unid15** [comment]
Chen packs are empty, its just the vars

**Comment by Shadowmech88** [comment]
Make sure to change the vars of the chempack so its safeties are disabled.
That is, if they drop them on death like I assume they do.

File: code/modules/mob/living/simple_animal/hostile/human/cult.dm [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+/obj/effect/landmark/corpse/cult
+	name = "cultist"
+	corpseshoes = /obj/item/clothing/shoes/cult
+	corpsesuit = /obj/item/clothing/suit/cultrobes
+	corpsemask = /obj/item/clothing/mask/gas/death_commando
+	corpseback = /obj/item/weapon/storage/backpack/cultpack
+
+/obj/effect/landmark/corpse/cult/New()
+	corpseuniform = existing_typesof(/obj/item/clothing/under/color)
```
Comment (clusterfack) [inline code/modules/mob/living/simple_animal/hostile/human/cult.dm]: Hm shit, did you mean to use pick(existing_typesof(/obj/item/clothing/under/color)) here?

Comment (unid15) [inline code/modules/mob/living/simple_animal/hostile/human/cult.dm:94]: Nope, these variables can be both type paths and lists

**Pull request merged**